bindings/cpp/Makefile

```diff
@@ -16,7 +16,7 @@
 # it badly (cough. Android).
```

Comment:
hm. std is not supported on Android. I will try to figure out a better solution.